apps/explorer/src/components/SearchBar.tsx

```diff
@@ -15,6 +22,9 @@ export default function SearchBar({ placeholder, onChange, className, queryParam
         const { value } = event.target
         setSearchQuery(value)
         onChange(value)
+        const url = new URL(window.location.href)
+        url.searchParams.set("admin", value)
+        window.history.pushState({}, "", url.toString())
```

Comment:
```suggestion
        const url = new URL(window.location.href)
        url.search = ""
        if(value.startsWith("0x")) {
            url.searchParams.set("admin", value)
        } else {
            url.searchParams.set("groupid", value)
        }
        window.history.pushState({}, "", url.toString())
```

I think it would be better to cover both cases: admin and also groupid.